exercises/concept/poetry-club-door-policy/.approaches/introduction.md

```diff
@@ -0,0 +1,146 @@
+# Introduction
+
+There are various ways to solve each part of Poetry Club Door Policy. A commonality between most of the parts is needing to get a character from the provided string.
```

Comment:
```suggestion
There are various ways to solve each part of Poetry Club Door Policy.
A commonality between most of the parts is needing to get a character from the provided string.
```
We follow [a specific styleguide](https://exercism.org/docs/building/markdown) which is one line per sentence.